_reports/shrikrishna.md

```diff
@@ -0,0 +1,248 @@
+---
+categories:
+  - GSoC
+  - 2018
+  - StatusReport
+date: 2018-08-12
+student: Shrikrishna Singh
+organisation: coala
+organisation_link: https://coala.io
+report_year: 2018
+project: Newcomer Metrics and Gamification
+project_link: https://summerofcode.withgoogle.com/projects/#5892040252981248
+tarball: https://github.com/sks444/GSoC-18-Patches/raw/master/project.tar.gz
+sha256sum: f5b22d4d8a8fe0d24f322c65c633e0f1304ac3f1e6f9cc0fb105d6e1a04f1278
+mentors: >
+  [John Vandenberg](http://github.com/jayvdb),
+  [Pranav Dixit](https://github.com/prnvdixit),
+  [Andrew Dassonville](https://github.com/andrewda) &
+  [Gopala](https://github.com/gkrishnan724)
+phase:
+  - Bonding : <https://gitlab.com/coala/GSoC/GSoC-2018/milestones/2>
+  - Phase 1 : <https://gitlab.com/coala/GSoC/GSoC-2018/milestones/19>
+  - Phase 2 : <https://gitlab.com/coala/GSoC/GSoC-2018/milestones/43>
+  - Phase 3 : <https://gitlab.com/coala/GSoC/GSoC-2018/milestones/44>
+bio: >
+  I am a final year Computer Science and Engineering undergraduate at
+  Shri Mata Vaishno Devi University. As part of the GSoC'18 I worked on
+  [community](https://github.com/coala/community) and
+  [webservices](https://gitlab.com/coala/landing/) project to build a
+  gamification system for all the active newcomers at coala.
+
+social:
+  - GitHub:
+    - username: sks444
+    - link: https://github.com/sks444
+  - GitLab:
+    - username: sks444
+    - link: https://gitlab.com/sks444
+  - Gitter:
+    - username: sks444
+    - link: https://gitter.im/sks444
+  - OpenHub:
+    - username: sks444
+    - link: https://www.openhub.net/accounts/sks444
+email: krishnasingh.ss30@gmail.com
+blog: https://sks444.github.io
+activity:
+  - 1:
+    - repo: coala-bears
+    - link: https://github.com/coala/coala-bears/commit/d35238fd
+    - details: >
+      GitCommitBear: Parse remote with giturlparse
```

Comment:
You'll have to add indents in all descriptions
Without an indent jekyll will throw errors